src/internal/Tile_getrf.hh

```diff
@@ -164,7 +171,8 @@ void getrf(
     std::vector<int64_t>& max_index,
     std::vector<int64_t>& max_offset,
     std::vector<scalar_t>& top_block,
-    blas::real_type<scalar_t> pivot_threshold)
+    blas::real_type<scalar_t> pivot_threshold,
+    int64_t* info )
```

Comment:
Because this gets called by every thread in the panel; only the thread with the diagonal element updates `info`. If it was an output parameter, the calling routine, `internal::getrf_panel`, would have to do a reduction across all the threads.

Reply:
That makes sense.